server/@types/activities.ts

```diff
@@ -17,21 +12,6 @@ export type Prisoner = {
   category?: string
 }
 
-export type ActivityScheduleAllocation = {
```

Comment:
Can't see that `ActivityScheduleAllocation` and `AllocationsSummary` are used any more